### PR TITLE
Set up application structure for admin client

### PR DESCRIPTION
**Pull request by lukeperrin10**
PT Story: https://www.pivotaltracker.com/story/show/178091741

## User Story 

```
As a development team,
In order to be able to start adding features,
We would like to have a basic application structure.
```

## Suggested Changes 
- Uses React JS as base library for client side application 

## Dependencies 
- Yarn

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```
Comment (emtalen) [inline src/App.jsx]: refactor this to an arrow function

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
```
Comment (emtalen) [inline src/index.css]: remove this file for now

**Review by emtalen — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (lukeperrin10) [reply src/App.jsx]: Refactored

File: src/index.css (hunk 2 above)
Comment (lukeperrin10) [reply src/index.css]: Has now been removed.

**Review by emtalen — APPROVED** [review]

**Pull request merged**